rust/parser/test/mod.rs

```diff
@@ -40,7 +40,7 @@ fn test_simple_query() {
 $x isa movie;"#;
 
     let parsed = parse_query(query).unwrap();
-    let expected = typeql_match(var("x").isa("movie"));
+    let expected = typeql_match(var("x").isa("movie")).unwrap();
     assert_query_eq!(expected, parsed, query);
```

Comment:
Pending finished error propagation, ideally `unwrap`s would be gone from most tests, and `assert_query_eq!` would handle that for us.